kubecost/templates/network-costs/network-costs-cluster-role.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if (.Values.networkCosts).enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "kubecost.networkCosts.name" . }}-network-costs-cluster-role
+  labels:
+    {{- include "kubecost.commonLabels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - nodes
+      - pods
+      - services
+      - endpoints
```

Comment:
I cross-checked these resources against:
- the original ClusterRole config which was known WAI for the network costs daemonset: https://github.com/kubecost/kubecost/blob/v2.8/cost-analyzer/templates/cost-analyzer-cluster-role-template.yaml
- analysis run by Bob against the contents of the https://github.com/kubecost/network-costs-rs repository 

Should any rules be missing from this ClusterRole, please let me know. Otherwise we will need to double check these rules are sufficient in a real install